tkupchyn/homework_05/non_unique_elements.py

```diff
@@ -15,5 +15,5 @@ def remove_unique_values(iterable: list) -> Iterable:
     return (elem for elem in iterable if counts[elem] > 1)
 
 
-def remove_unique_values_alternative(iterable: list) -> Iterable:
+def remove_unique_values_alternative(iterable: list) -> Generator:
```

Comment:
try to avoid changes if they do not relate to this pr, 
and move them to sepparate one